src/modules/EventList/index.tsx

```diff
@@ -602,7 +602,26 @@ export const EventList = ({ useConfigOptions = false }: EventListProps): ReactEl
     const onSelectEvent = useCallback<ChangeHandler<EventListEvent>>(
         (event, change) => {
             const primaryQuote = getPrimaryQuote(change.value?.primaryCompany);
+            const ticker = primaryQuote?.localTicker;
+            const title = change.value?.title;
+            const eventDate = change.value?.eventDate;
+            const eventType = change.value?.eventType;
             bus?.emit('instrument-selected', { ticker: primaryQuote?.localTicker }, 'out');
+
+            // We don't need to emit events
+            // when going back to the event list
+            if (ticker && title) {
```

Comment:
technically this wouldn't broadcast if a private event doesn't have a ticker... so maybe I should change this to be conditional on title and date? @mmichael0413